### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by AnthonyGlt**
2 issue templates:
- Bug <- The old issue template is used
- Proposal <- Template for new feature or ideas

### CODE REVIEW

**Review by jailln — APPROVED** [review]

**Comment by mgermerie** [comment]
Thanks for this addition. Wouldn't we need to delete the current [`.github/ISSUE_TEMPLATE.md`](https://github.com/iTowns/itowns/blob/master/.github/ISSUE_TEMPLATE.md) file as well ?

**Comment by AnthonyGlt** [comment]
We can keep it, it will be used in this case:
image
We could also edit it 
I'll try to remove it, I think an other template will be shown

File: .github/ISSUE_TEMPLATE/proposal-request.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the proposal in the Title above -->
+This issue is a **feature proposal**. Feel free to upvote (with :+1: ), comment and provide your use-cases if you're interested by this feature.
+
+## Context
+<!--- A description of the feature and why this feature could be interesting to implement in iTowns! -->
+<!--- Current state of iTowns vs. state after implementation -->
```
Comment (Desplandis) [inline .github/ISSUE_TEMPLATE/proposal-request.md:21]: The impacting changes part shall be in a dedicated part. See comment below.

File: .github/ISSUE_TEMPLATE/proposal-request.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
```
Comment (Desplandis) [inline .github/ISSUE_TEMPLATE/proposal-request.md:22]: We could include an *optional* sub-part describing the state-of-the-art of such functionality or its use in other tools.

File: .github/ISSUE_TEMPLATE/proposal-request.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Expected behavior
+<!--- What would this feature looks like in iTowns ? -->
+<!--- Please tell us what you are trying to accomplish -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+<!--- Could be screenshot of the feature implemented somewhere else or results/stats/poc showing the potential result -->
+
+## Proposal
```
Comment (Desplandis) [inline .github/ISSUE_TEMPLATE/proposal-request.md]: "Proposal" shall be renamed to "Implementation" and shall be an optional part if there is no proposed implementation. Add a comment (e.g. "Describe the proposed implementation (PR, pseudo-code and/or diagrams)").

The idea is to have 2 main parts of the proposal (a functional and technical part). The optional is mandatory (and IMHO the important part) and the technical optional but is mandatory if there is an implementation proposed/an open PR.

File: .github/ISSUE_TEMPLATE/proposal-request.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+<!--- Provide a general summary of the proposal in the Title above -->
+This issue is a **feature proposal**. Feel free to upvote (with :+1: ), comment and provide your use-cases if you're interested by this feature.
+
+## Context
+<!--- A description of the feature and why this feature could be interesting to implement in iTowns! -->
+<!--- Current state of iTowns vs. state after implementation -->
+
+
+## Expected behavior
```
Comment (Desplandis) [inline .github/ISSUE_TEMPLATE/proposal-request.md]: "Expected behavior" shall be renamed to "Description of the proposal" (in terms of functionnality).

File: .github/ISSUE_TEMPLATE/proposal-request.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+## Proposal
+<!--- Things that should be done in iTowns to implement this feature -->
+
+## Potential Problems
+<!--- Things that could be problematic with this feature or its implementation -->
+
+## Potential Solutions
+<!--- Things that could be done to avoid the mentioned problems -->
+
+## Identified use-cases
```
Comment (Desplandis) [inline .github/ISSUE_TEMPLATE/proposal-request.md]: Use cases shall be on top (below or as a sub-part of "Description of the Proposal")!
This list will be updated by the assigned person given the comments on the proposal.

File: .github/ISSUE_TEMPLATE/proposal-request.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+<!--- What would this feature looks like in iTowns ? -->
+<!--- Please tell us what you are trying to accomplish -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+<!--- Could be screenshot of the feature implemented somewhere else or results/stats/poc showing the potential result -->
+
+## Proposal
+<!--- Things that should be done in iTowns to implement this feature -->
+
+## Potential Problems
+<!--- Things that could be problematic with this feature or its implementation -->
```
Comment (Desplandis) [inline .github/ISSUE_TEMPLATE/proposal-request.md:42]: IMHO there could be multiple potential solutions/problems.

File: .github/ISSUE_TEMPLATE/proposal-request.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+## Potential Solutions
+<!--- Things that could be done to avoid the mentioned problems -->
+
+## Identified use-cases
+<!--- Define in which context this feature could be useful for the user -->
+
```
Comment (Desplandis) [inline .github/ISSUE_TEMPLATE/proposal-request.md]: If there is a proposed implementation, there should be a study of the impact (either by the person proposing or one of the maintainer) as a PR, pseudo-code and/or diagrams.

File: .github/ISSUE_TEMPLATE/proposal-request.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+name: Proposal request
+about: Suggest an idea for this project
+title: ''
+labels: 'proposal :+1:'
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the proposal in the Title above -->
+This issue is a **feature proposal**. Feel free to upvote (with :+1: ), comment and provide your use-cases if you're interested by this feature.
```
Comment (Desplandis) [inline .github/ISSUE_TEMPLATE/proposal-request.md:17]: We should add an **important** notice that a proposal = one and only one feature.

**Review by Desplandis — COMMENTED** [review]
Sorry for the wait, I added a few comments on the proposed proposal request!
We could discuss it in this issue. =)

**Pull request merged**